compiler/src/iree/compiler/ExternalInterfaces/UtilExternalModels.cpp

```diff
@@ -490,13 +492,24 @@ void registerUtilExternalModels(DialectRegistry &registry) {
 
   // Hoistable Op Interface registration.
 
-  // Register hoistable type interfaces for LinalgExt ops.
+  // Register hoistable op interfaces for Encoding ops.
   registry.addExtension(
       +[](MLIRContext *context, IREE::Encoding::IREEEncodingDialect *dialect) {
         UnhoistableOpInterfaceHelper<
             IREE::Encoding::SetEncodingOp>::registerOpInterface(context);
       });
-  // Register hoistable type interfaces for linalg ops.
+
+  // Register hoistable op interfaces for Flow ops.
+  registry.addExtension(
+      +[](MLIRContext *context, IREE::Flow::FlowDialect *dialect) {
+        UnhoistableOpInterfaceHelper<
+            IREE::Flow::DispatchWorkgroupCountOp>::registerOpInterface(context);
```

Comment:
It turns out that I just need to add the op to unhoistable list, which makes sense to me.